src/DRICH_geo.cpp

```diff
@@ -318,26 +379,29 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
       radiatorRmax +
           snoutDelta * (aerogelThickness + airgapThickness + filterThickness) / snoutLength);
 
-  Volume aerogelVol(detName + "_aerogel", aerogelSolid, aerogelMat);
+  //Volume aerogelVol(detName + "_aerogel", aerogelSolid, aerogelMat);
```

Comment:
Same here and elsewhere.

Reply:
Hi @veprbl Dimtry, yes I know. I wanted to put this draft PR in order to make sure our young collaborators, work directly on a branch that is under radar of the software experts. As it will be part of their PhD studies. But, my impression is, they haven't yet managed to get an dev-access. 
Hence I think I will take care of the overlap and other issues.